scripts/vote.coffee

```diff
@@ -128,9 +128,13 @@ Supported commands:
     msg.reply if Vote.current then sure_am else nope
 
   # Election driver
-  robot.respond /(?:vote|freedom|oppressed) ([^\s]+)\s?(.*)?$/, (msg) ->
-    action = msg.match[1].trim().toLowerCase()
-    arg = msg.match[2]?.trim()
+  robot.respond /(?:vote|freedom|oppressed) (\d+ )?([^\s]+)\s?(.*)?$/, (msg) ->
+    timeout = 1 # default timeout is 1 minute
+    if msg.match[1] and msg.match[1].length > 0
```

Comment:
Is it necessary to do the first comparison?

`if msg.match[1].length > 0`

I am ignorant to Javascript comparisons. So sorry if that is the best way!